patches/minecraft/net/minecraft/world/gen/layer/BiomeLayer.java.patch

```diff
@@ -0,0 +1,77 @@
+--- a/net/minecraft/world/gen/layer/BiomeLayer.java
```

Comment:
this patch can probably be cleaned up a bit since we're readding it

Reply:
Can you be more specific?
I would like to keep the structure as closely to Vanilla as reasonable.
Could probably get rid of the null checks (and use empty lists in the biome manager where necessary) though.
Could move the legacy biome stuff into BiomeManager as well, but I think it is more appropriate here as it shouldn't be modified by modders